database/dataGeneration/quoteInfo/cluster.js

```diff
@@ -0,0 +1,56 @@
+const cluster = require('cluster');
+let numCPUs = require('os').cpus().length;
+const generateData = require('./quoteGenerator.js');
+
+numCPUs = 8;
```

Comment:
It may be a good idea to obtain this through a library, as the computers on AWS may require a different numCPUs here.